.github/workflows/checklist.yml

```
@@ -0,0 +1,16 @@
+name: Add comment
+
+on:
+- pull_request_target
+
+jobs:
+  comment:
+    name: Checklist
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v0.1.0
```

Why is this one using such an old version of checkout?
```suggestion
    - uses: actions/checkout@v3
```

---

Oops!  That was supposed to be for the version of `harupy/comment-on-pr`.